GCD_HomeWork/GCD_HomeWork/DetailViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  DetailViewController.swift
+//  GCD_HomeWork
+//
+//  Created by Bach Nghiem on 30/08/2023.
+//
+
+import UIKit
+
+final class DetailViewController: UIViewController {
+    
+    @IBOutlet private weak var nameLabel: UILabel!
+    @IBOutlet private weak var userImage: UIImageView!
+    @IBOutlet private weak var descriptionLabel: UILabel!
+    @IBOutlet private weak var tableView: UITableView!
+    @IBOutlet private weak var segmentedControl: UISegmentedControl!
+    @IBOutlet private weak var stackView: UIStackView!
+    @IBOutlet private weak var backButton: UIButton!
+    @IBOutlet private weak var favouriteButton: UIButton!
+    
+    private var users = [User]()
+    var user : User?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        if let user = user {
+            userImage.image = UIImage(named: user.avatar)
+            nameLabel.text = user.name
+            descriptionLabel.text = user.reposURL
+        }
+        config()
+        getUsers()
+    }
+    
+    @IBAction private func handleFavouriteButton(_ sender: Any) {
+        
+    }
+    
+    @IBAction private func handleBackButton(_ sender: Any) {
+        navigationController?.popViewController(animated: true)
+    }
+    
+    private func config() {
+        navigationController?.isNavigationBarHidden = true
+        favouriteButton.radius()
+        tableView.dataSource = self
+        tableView.delegate = self
+        userImage.radius()
+        stackView.borderViewRadius(radius: LayoutOptions.radius)
+    }
+    
+    private func getUsers() {
+        let userArr = [User(avatar: "test", name: "Username", reposURL: "github.com/abcde"),
+                       User(avatar: "test", name: "Username", reposURL: "github.com/123"),
+                       User(avatar: "test", name: "Username", reposURL: "github.com/jqkl")
+        ]
+        _ = userArr.map { users.append($0) }
```

Comment:
như này là được rồi e nhỉ ?, đổi tên lại `initUsers()` luôn e
```suggestion
        users = [User(avatar: "test", name: "Username", reposURL: "github.com/abcde"),
                       User(avatar: "test", name: "Username", reposURL: "github.com/123"),
                       User(avatar: "test", name: "Username", reposURL: "github.com/jqkl")]
```